src/main/java/edu/ie3/datamodel/io/source/sql/SqlTimeSeriesMappingSource.java

```diff
@@ -43,13 +47,28 @@
         .collect(Collectors.toMap(MappingEntry::getParticipant, MappingEntry::getTimeSeries));
   }
 
+  /** @deprecated since 3.0. Use {@link #timeSeriesMetaInformation(java.util.UUID)} instead */
   @Override
-  public Optional<IndividualTimeSeriesMetaInformation> getTimeSeriesMetaInformation(
-      UUID timeSeriesUuid) {
+  @Deprecated(since = "3.0", forRemoval = true)
+  public Optional<edu.ie3.datamodel.io.csv.timeseries.IndividualTimeSeriesMetaInformation>
+      getTimeSeriesMetaInformation(UUID timeSeriesUuid) {
     return getDbTableName(null, "%" + timeSeriesUuid.toString())
         .map(entityPersistenceNamingStrategy::extractIndividualTimesSeriesMetaInformation);
   }
 
+  @Override
+  public Optional<IndividualTimeSeriesMetaInformation> timeSeriesMetaInformation(
```

Comment:
The deprecated predecessor method `getTimeSeriesMetaInformation` has a different logic than this (it does not account for the schema name). I'd like you to consider altering the existing method as well.

Reply:
I didn't want to do this at first because it changes the functionality of an existing method, but since the whole SQL timeseries stuff has been introduced within this version anyway, it's fine